### PR TITLE
New Stack Sprites

### PR DESCRIPTION
**Pull request by ghost**
New stack sprites for wood, plasteel, metal, glass, rglass, rods by @Partheo
sprite
sprite2

:cl: Partheo
imageadd: Wood, plasteel, metal, glass, rglass, rods have new sprites
/:cl:

### CODE REVIEW

**Comment by DaedalusGame** [comment]
where does the fucking tape come from

**Comment by uraniummeltdown** [comment]
nanobots

**Comment by DaedalusGame** [comment]
i will only accept this change if the tape goes away

**Comment by praisenarsie** [comment]
these were pr'd before and they were closed

**Comment by DaedalusGame** [comment]
actually no i don't accept these at all because don't let you hide items underneath sheets take this to CM reeee

**Comment by vuonojenmustaturska** [comment]
These look good enough to me, the perspective is a bit odd though

**Comment by Iamgoofball** [comment]
these are sexy ignore daedalus whining about edge case shit

**Comment by Ausops** [comment]
they look really nice but can you rotate them by 45 degrees so they fit in the 3/4 perspective we have while still showing off 3 sides

**Comment by DaedalusGame** [comment]
>showing three sides in 3/4 perspective

**Comment by Tlaltecuhtli** [comment]
from where did u steal it

**Comment by praisenarsie** [comment]
yog

**Comment by Bawhoppen** [comment]
I don't like these ones. They look bland and *washed out* for a lack of a better term. This PR also doesn't update titanium/plastitanium or any of the special glass types to the new sheet style either.

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -57,14 +57,16 @@
 /obj/item/stack/update_icon()
 	if(novariants)
 		return ..()
+	icon_amount_update()
+	..()
```
Comment (ShizCalev) [inline code/game/objects/items/stacks/stack.dm]: this ..() doesn't do anything

File: code/game/objects/items/stacks/stack.dm [hunk 2]
```diff
@@ -57,14 +57,16 @@
 /obj/item/stack/update_icon()
 	if(novariants)
 		return ..()
```
Comment (ShizCalev) [inline code/game/objects/items/stacks/stack.dm]: nor does this one

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by WJohn — CHANGES_REQUESTED** [review]
Needs to be changed from oblique to isometric, plasteel needs work to be given a more clearly visible bump. 

A bit of roundness to the corners would also go a long way, as these have very sharp edges (I feel it's a bit less interesting than what we have now). Plasteel can keep the sharp edges.

The metalic sheen should be reworked a bit (which is likely gonna happen with the perspective change).

Also please apply this to the other stack objects like plasma glass, titanium, plastic, and so on.

**Comment by Kawaii-Big-Boss** [comment]
shit this actually looks pretty good, really like the cable wraps

**Comment by uraniummeltdown** [comment]
I have no experience in spriting so it'll be impossible for me to resprite these in another perspective, let alone do plastic and friends

**Comment by Iamgoofball** [comment]
@WJohn can you take a stab at this

**Pull request closed without merging**